python/amici/ode_export.py

```diff
@@ -2310,9 +2338,13 @@ def _writeModelHeader(self):
             'NEVENT': '0',
             'NOBJECTIVE': '1',
             'NW': str(len(self.model.sym('w'))),
-            'NDWDP': str(len(self.model.eq('dwdp'))),
+            'NDWDP': str(len(self.model.sparsesym('dwdp'))),
             'NDWDX': str(len(self.model.sparsesym('dwdx'))),
             'NDXDOTDW': str(len(self.model.sparsesym('dxdotdw'))),
+            'NDXDOTDP_EXPLICIT': str(len(self.model.sparsesym(
```

Comment:
why does `self.model.sparsesym('dxdotdp')` even exist? All the specific cases above suggest a similar mismatch between `dxdotdp_explicit` and `dxdotdp`. Is there any specific reason for that? Why not remove `dxdotdp` completely and always go with `dxdotdp_explicit`?

Reply:
Reason was: This way, I could directly use the implemented code for differentiating xdot w.r.t p by keeping dxdotdp, without writing too many new, additional if-clauses... It was the best way to make it work for me, since this meant rather few changes in ode_export.py. I wanted to avoid doing too many changes here, as ode_export is far from being easily understandable, with recursive loops between function calls...